atom_batch_execution/scripts/plot_graphs

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env python3
+"""
+Plots processed results from a batch given a yaml file for configuration
+"""
+import argparse
+import glob
+import os
+import shutil
+from os.path import isfile
+import yaml
+from pprint import pprint
+import csv
+
+from colorama import Fore, Style
+import matplotlib.pyplot as plt
+from matplotlib.lines import Line2D
```

Comment:
I heavily suggest restructuring this code to the seaborn library. It is based on matplotlib but enhances the user experience and allows for better looking graphs.

Reply:
I remember you telling me this. The plotting script  is somewhat independent of the plotting library used. It could adapted somewhat easily if I was familiar with the library. However I'm not familiar with seaborn and this script is not actively being used so I don't think its a priority.

I will however revisit your suggestion in the future when I have the need to use this plotting script.